src/applications/check-in/tests/e2e/pages/NextOfKin.js

```diff
@@ -49,16 +49,22 @@ class NextOfKin {
 
   validateAdditionalInfo = {
     dayOf: () => {
-      cy.get('div[data-testid="additional-info"]').should(
-        'contain.text',
-        'If this isn’t your correct information, select No and a staff member can help you update your information on the day of your appointment.',
-      );
+      this.openAdditionalInfo();
+      cy.get('div[data-testid="additional-info"]')
+        .should('be.visible')
+        .should(
+          'contain.text',
+          'If this isn’t your correct information, select No and a staff member can help you check in and update your information.',
+        );
     },
     preCheckIn: () => {
-      cy.get('div[data-testid="additional-info"]').should(
-        'contain.text',
-        'If this isn’t your correct information, select No and a staff member can help you update your information on the day of your appointment.',
-      );
+      this.openAdditionalInfo();
+      cy.get('div[data-testid="additional-info"]')
+        .should('be.visible')
```

Comment:
For the `be.visible` asserts we should check the visibility on a div or something within it that is visible after you open the drawer.

Reply:
True, gotcha